README.md

```diff
@@ -12,77 +12,71 @@ This package contains a set of tools to load and visualize the
-distribution) or `pip install jupyter` (for the other Python distributions)
-
-NB: For [NERSC](http://www.nersc.gov/) users, it is not necessary to
-install the above package, as NERSC provides it when logging to
-[https://ipython.nersc.gov](https://ipython.nersc.gov).
-Therefore, NERSC users only need to install the `openPMD-viewer`
-package itself.
-
-## Usage
-
-The routines of openPMD-viewer can be used in two ways :
+The routines of `openPMD-viewer` can be used in two ways :
 
 - Use the **Python API**, in order to write a script that loads the
   data and produces a set of pre-defined plots.
 
-- Use the **interactive GUI inside the IPython Notebook**, in order to interactively
+- Use the **interactive GUI inside the Jupyter Notebook**, in order to interactively
 visualize the data.
 
+## Usage
+
 ### Tutorials
 
 The notebooks in the folder `tutorials/` demonstrate how to use both
 the API and the interactive GUI. You can view these notebooks online
-[here](https://github.com/openPMD/openPMD-viewer/tree/master/tutorials),
-or, alternatively, you can run them on your local computer by typing:
-
-`ipython notebook tutorials/`
+[here](https://github.com/openPMD/openPMD-viewer/tree/master/tutorials).
 
-NB: For [NERSC](http://www.nersc.gov/) users, you can run the tutorials on a
-remote machine by logging in at
-[https://ipython.nersc.gov](https://ipython.nersc.gov), and by
-navigating to your personal copy of the directory `openPMD-viewer/tutorials`.
+You can also download and run these notebooks on your local computer
+(when viewing the notebooks with the above link, click on `Raw` to be able to
+save them to your local computer).
 
 ### Notebook quick-starter
 
-If you wish to use the **interactive GUI**, the installation of `openPMD-viewer` provides
-a convenient executable which automatically
+If you wish to use the **interactive GUI**, the installation of
+`openPMD-viewer` provides a convenient executable which automatically
 **creates a new pre-filled notebook** and **opens it in a
 browser**. To use this executable, simply type in a regular terminal:
 
 `openPMD_notebook`
 
-(This executable is installed by default when running `python setup.py install`.)
+(This executable is installed by default.)
+
+## Installation
+
+### Installation on a local computer, using Anaconda
+
+On a local computer, it is recommended to install the [Anaconda
+distribution](https://www.continuum.io/downloads), which enables an easy
+installation of the dependencies of `openPMD-viewer`.
+
+Once Anaconda is installed, type
+```
+conda install scipy matplolib h5py jupyter
+pip install openPMD-viewer
+```
+The first line installs the dependencies, while the second line
+installs `openPMD-viewer` itself
+
+### Installation on a remote scientific cluster
+
+If you wish to install the `openPMD-viewer` on a remote scientific
+cluster, please make sure that packages `numpy`, `scipy` and `h5py`
+are available in your environment. This is typically done by a set of

```

Comment:
to be honest: the module load python packages do not work 95% of the time due to ABI incompatebilities with various compilers that are common.

Due to that many sysadmins deprecate to even provide more than a plain python and encourage to go for venvs in user mode.